src/extension.ts

```diff
@@ -142,6 +142,14 @@ export function activate(context: vscode.ExtensionContext) {
 				await vscode.commands.executeCommand("cosmosDB.refresh");
 			}
 		});
+	registerCommand('cosmosDB.api.revealTreeItem', async (treeItemId: string) => {
+		const customView = vscode.window.createTreeView('cosmosDBExplorer', { treeDataProvider: tree });
+		const node = await tree.findTreeItem(treeItemId);
+		if (!node) {
```

Comment:
Please move all this code to a separate file here: https://github.com/Microsoft/vscode-cosmosdb/tree/master/src/commands

The Cosmos DB repo is the worst example, but I'm slowly trying to convert us over to that pattern.